critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -13,19 +13,25 @@
   <div class="row">
     {% include ('review/modify/%s.html' % entity_type) %}
   </div>
-  <div class="clearfix">
-    <ul class="nav nav-tabs" style="margin-bottom:-1px;">
-      <li class="active"><a href="#write" data-toggle="tab">{{ _('Write') }}</a></li>
-      <li><a href="#preview" data-toggle="tab">{{ _('Preview') }}</a></li>
-      <small class="text-muted pull-right" style="margin-top:20px;font-style:italic;">
-        {{ _('You can use <a href="%(url)s" target="_blank">Markdown</a> syntax to apply custom formatting.',
-              url='http://daringfireball.net/projects/markdown/syntax') }}
-      </small>
-    </ul>
-  </div>
+  <hr />
+
   <form id="review-editor" method="POST" class="form-horizontal" role="form">
     {{ form.hidden_tag() }}
     <div class="form-group">
+      <label class="col-sm-2" for="rating">{{ _('Rating for the review:') }}</label>
```

Comment:
`for the review` part can be removed since it's pretty obvious what the rating is for.

Reply:
Removed it and changed the column grid accordingly.